### PR TITLE
Fix reset calculation in forward function

### PR DESCRIPTION
**Pull request by rG223**
Hi, I found an issue in tutorial 6.
```
  def forward(self, input_, mem):
    spk = self.surrogate_func((mem-self.threshold))  # call the Heaviside function
    reset = (spk - self.threshold).detach()
    mem = self.beta * mem + input_ - reset
    return spk, mem
```
The reset computation is incorrect, and it should be `reset = (spk * self.threshold).detach()`.

### CODE REVIEW

**Comment by ixfd64** [comment]
I applied the change to the Chinese version as well.

CI failures are unrelated to this change.
